[0m[[0m[31merror[0m] [0m[0mjava.lang.IllegalArgumentException: Argument unrecognized by ScalaTest's Runner: a[0m
[0m[[0m[31merror[0m] [0m[0m	at org.scalatest.tools.ArgsParser$.parseArgs(ArgsParser.scala:450)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.scalatest.tools.Framework.runner(Framework.scala:960)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Defaults$.$anonfun$createTestRunners$1(Defaults.scala:1099)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:238)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.collection.immutable.Map$Map2.foreach(Map.scala:159)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.collection.TraversableLike.map(TraversableLike.scala:238)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.collection.TraversableLike.map$(TraversableLike.scala:231)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.collection.AbstractTraversable.map(Traversable.scala:108)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Defaults$.createTestRunners(Defaults.scala:1093)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Defaults$.allTestGroupsTask(Defaults.scala:1161)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Defaults$.$anonfun$inputTests0$4(Defaults.scala:1078)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.std.FullInstance$.$anonfun$flattenFun$3(TaskMacro.scala:80)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.Function1.$anonfun$compose$1(Function1.scala:49)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.std.Transform$$anon$4.work(Transform.scala:67)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Execute.$anonfun$submit$2(Execute.scala:281)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:19)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Execute.work(Execute.scala:290)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Execute.$anonfun$submit$1(Execute.scala:281)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:178)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.CompletionService$$anon$2.call(CompletionService.scala:37)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.util.concurrent.FutureTask.run(FutureTask.java:266)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.util.concurrent.FutureTask.run(FutureTask.java:266)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.lang.Thread.run(Thread.java:748)[0m
[0m[[0m[31merror[0m] [0m[0m(Test / [31mtestOnly[0m) java.lang.IllegalArgumentException: Argument unrecognized by ScalaTest's Runner: a[0m
